File: TransCoder_tokenized_test_set_functions/python/PRINT_STRING_SPECIFIED_CHARACTER_OCCURRED_GIVEN_NO_TIMES.py
def printString ( str, ch, count ) :
    occ, i = 0, 0
    if ( count == 0 ) :
        print ( str )
    for i in range ( len ( str ) ) :
        if ( str [ i ] == ch ) :
            occ += 1
        if ( occ == count ) :
            break
    if ( i < len ( str ) - 1 ) :
        print ( str [ i + 1 : len ( str ) - i + 2 ] )
    else :
        print ( "Empty string" )
    

File: TransCoder_tokenized_test_set_functions/cpp/PRINT_STRING_SPECIFIED_CHARACTER_OCCURRED_GIVEN_NO_TIMES.cpp
void printString ( string str, char ch, int count ) {
  int occ = 0, i;
  if ( count == 0 ) {
    cout << str;
    return;
  }
  for ( i = 0;
  i < str . length ( );
  i ++ ) {
    if ( str [ i ] == ch ) occ ++;
    if ( occ == count ) break;
  }
  if ( i < str . length ( ) - 1 ) cout << str . substr ( i + 1, str . length ( ) - ( i + 1 ) );
  else cout << "Empty string";
}